[CI] ParquetTestingIT testParquetFile {bad_ARROW-GH-41321} failing

**Build Scans:**
- [elasticsearch-intake #40602 / part5](https://gradle-enterprise.elastic.co/s/izphpx7scvbek)
- [elasticsearch-pull-request #150976 / part-5](https://gradle-enterprise.elastic.co/s/copxasuqornuu)
- [elasticsearch-pull-request #150369 / part-5](https://gradle-enterprise.elastic.co/s/zkngjgw4ivuo4)
- [elasticsearch-pull-request #150369 / part-5](https://gradle-enterprise.elastic.co/s/6klmaxrcvehm4)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql-datasource-parquet:qa:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.parquet.ParquetTestingIT.testParquetFile {bad_ARROW-GH-41321}" -Dtests.seed=BF851F70EB2BA197 -Dtests.locale=bm-ML -Dtests.timezone=Africa/Cairo -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.parquet.ParquetTestingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testParquetFile%20%7Bbad_ARROW-GH-41321%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Bad data file bad_data/ARROW-GH-41321.parquet should produce a 4xx error but got 500: method [POST], host [http://[::1]:43203], URI [/_query?format=yaml&pretty=true&error_trace=true], status line [HTTP/1.1 500 Internal Server Error]
---
error:
  root_cause:
  - type: "parquet_decoding_exception"
    reason: "could not read page Page [bytes.size=16, valueCount=4, uncompressedSize=16]\
      \ in col [fixed_size_list_float64, list, item] optional double item"
    stack_trace: "org.elasticsearch.ElasticsearchException$1: could not read page\
      \ Page [bytes.size=16, valueCount=4, uncompressedSize=16] in col [fixed_size_list_float64,\
      \ list, item] optional double item\n\tat org.elasticsearch.server@9.5.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:811)\n\
      \tat org.elasticsearch.server@9.5.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:725)\n\
[truncated]
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testParquetFile {bad_ARROW-GH-41321}
- [main] 1 consecutive failures in step part5
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 4 failures in test testParquetFile {bad_ARROW-GH-41321} (66.7% fail rate in 6 executions)
- [main] 3 failures in step part-5 (75.0% fail rate in 4 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (100.0% fail rate in 2 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -532,9 +532,6 @@ tests:
 - class: org.elasticsearch.xpack.stateless.engine.translog.StatelessTranslogIT
   method: testTranslogStressRecoveryTest
   issue: https://github.com/elastic/elasticsearch/issues/150297
-- class: org.elasticsearch.xpack.esql.qa.parquet.ParquetTestingIT
-  method: testParquetFile {bad_ARROW-GH-41321}
-  issue: https://github.com/elastic/elasticsearch/issues/150300
 - class: org.elasticsearch.repositories.gcs.GoogleCloudStorageBlobContainerRetriesTests
   method: testWriteLargeBlob
   issue: https://github.com/elastic/elasticsearch/issues/150356

